Include/internal/pycore_bitutils.h

```diff
@@ -131,8 +131,32 @@ _Py_popcount32(uint32_t x)
 }
 
 
+// Return the index of the most significant 1 bit in 'x'. This is the smallest
+// integer k such that x < 2**k. Equivalent to floor(log2(x)) + 1 for x != 0.
+static inline int
+_Py_bit_length(unsigned long x) {
+    if (!x) {
+        return 0;
+    }
+#if (defined(__clang__) || defined(__GNUC__))
```

Comment:
In your previous PR, you wrote:

```
 #if defined(__GNUC__) && (((__GNUC__ == 3) && (__GNUC_MINOR__ >= 4)) || (__GNUC__ >= 4)) 
```

I'm not sure that we do need to support GCC older than GCC 4. But maybe just add a comment to document that it was added to GCC 3.4.